src/components/pages/Configure.tsx

```diff
@@ -0,0 +1,182 @@
+        acc[curr.name] = curr.value;
+        return acc;
+      }, {})
+    );
+  };
+
+  useEffect(() => {
+    if (!!notebookMetadata) {
+      loadConfigFromMetadata();
+    }
+  }, [notebookMetadata]);
+
+  const configBundles = UIStore.useState(s => s.configBundleOptions);
+  const configuredOptionsFromBundle = useMemo(() => {
+    if (!cluster) return {};
+
+    const map: any = {};
+    configBundles
+      .filter(b => selectedConfigBundles.includes(b.name))
+      .filter(b => !b.clusterFilter || b.clusterFilter.includes(cluster.value))
+      .forEach(bundle => {
+        bundle.options.forEach(option => {
+          if (!map[option.name] || option.concatenate === undefined) {
+            map[option.name] = option.value;
+          } else {
+            map[option.name] = `${map[option.name] ?? ''}${option.concatenate}${option.value}`;
+          }
+        });
+      });
+    return map;
+  }, [configBundles, cluster?.value, selectedConfigBundles]);
+
+  // Note: existing config will be overriden WITHOUT concatenation
+  const configuredOptions: { [key: string]: any } = { ...configuredOptionsFromBundle, ...extraConfig };
+
+  const reviewConfiguration = () => {
+    showDialog({
+      title: 'Review configuration',
+      buttons: [
+        {
+          label: 'Close',
+          caption: 'Close dialog',
+          className: '',
+          accept: false,
+          displayType: 'default',
+          ariaLabel: '',
+          iconClass: '',
+          iconLabel: '',
+          actions: []
+        }
+      ],
+      body: (
+        <div style={{ display: 'flex', flexDirection: 'column', gap: 8, width: 400 }}>
+          {Object.keys(configuredOptions).length === 0 && <div>No configuration</div>}
+          {Object.keys(configuredOptions).map(option => (
+            <div>
+              <div>{option}</div>
+              <small style={{ color: 'var(--jp-ui-font-color2)' }}>{configuredOptions[option]}</small>
+            </div>
+          ))}
+        </div>
+      )
+    });
+  };
+
+  const { mutate } = useStatus();
+  const connect = () => {
+    UIStore.update(s => {
+      s.isConnecting = true;
+      s.clusterName = cluster?.value;
+    });
+
+    const requestBody = {
+      cluster: cluster?.value,
+      options: configuredOptions,
+      configBundles: selectedConfigBundles,
+      extraConfig: extraConfig
+    };
+
+    requestAPI<any>('/cluster/start', {
+      method: 'POST',
+      body: JSON.stringify(requestBody),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then(mutate)
+      .finally(() => {
+        UIStore.update(s => {
+          s.isConnecting = false;
+        });
+      });
+  };
+
+  return (
+    <div style={{ flex: 1, overflow: 'auto', display: 'flex', flexDirection: 'column', alignItems: 'stretch' }}>
+      <div style={{ padding: 8 }}>
+        <SparkLogo />
+        <h3 className="jp-SparkConnectExtension-heading">Connect to Cluster</h3>
```

Comment:
I think "Connect to Cluster" is appropriate, because the title should refer to the "connect" page as opposed to the whole extension.

However, I think it's possible to add "Initialize a shared Spark Server instance using cluster resources." as a subtitle. What do you think?

Reply:
Something like this:
<img width="307" alt="Screenshot 2023-09-01 at 11 29 33" src="https://github.com/swan-cern/sparkconnect/assets/4845621/56b6e880-a43c-467e-8400-14fc53265f37">